Formula/wxmac.rb

```diff
@@ -13,23 +13,27 @@ class Wxmac < Formula
     sha256 "591bffb395156566ef3e55a9a8f2c193bdcde8b941d42cc6e6b41833ceeeaba2" => :yosemite
   end
 
+  # Fix compilation on High Sierra
+  # https://trac.wxwidgets.org/ticket/17929#ticket
+  patch :DATA
```

Comment:
I didn't think the commit would have the regenerated file, but it does!

Patch can be included on devel, it doesn't hurt.